src/Service/PaymentsService.php

```diff
@@ -80,4 +80,14 @@ public function refundPayment($id, $amount)
 		return $this->makeRequest('POST', 'payments/payment/' . $id . '/refund', ['amount' => round($amount * 100)], Http::CONTENT_FORM);
 	}
 
+	/**
+     * @param string $currency
+     * @return Response
+     */
+    public function getEnabledInstruments($currency)
```

Comment:
Maybe it's better. As I see it, all of method names in this class contains word `payment`, so I will rename it.